qiskit/extensions/standard/cxbase.py

```diff
@@ -16,18 +16,18 @@
 Fundamental controlled-NOT gate.
 """
 import warnings
-from qiskit.circuit import Gate
+from qiskit.circuit import ControlledGate
 from qiskit.circuit import QuantumCircuit
 
 
-class CXBase(Gate):
+class CXBase(ControlledGate):
     """Fundamental controlled-NOT gate."""
 
     def __init__(self):
         """Create new CX instruction."""
         warnings.warn('CXBase is deprecated and it will be removed after 0.9. '
                       'Use CnotGate instead.', DeprecationWarning, 2)
-        super().__init__("CX", 2, [])
+        super().__init__("CX", 2, [], num_ctrl_qubits=1)
 
```

Comment:
add `self.base_gate` here for the cxbase gate?

Reply:
Done